lineman/app/models/user_model.coffee

```diff
@@ -57,6 +57,17 @@ angular.module('loomioApp').factory 'UserModel', (BaseModel) ->
       proposal.isActive() and
       (@isAdminOf(proposal.group()) or @isAuthorOf(proposal))
 
+    canCreateOutcomeFor: (proposal) ->
```

Comment:
This seems like it can be unduplicated some... and it seems like proposal.isClosed() should be on both, no?

```
proposal.hasOutcome() and @canSetOutcomeFor(proposal)
```
and
```
!proposal.hasOutcome() and @canSetOutcomeFor(proposal)
```